app/client/router/routerApplications.js

```diff
@@ -10,14 +10,14 @@ Router.map(function() {
 
         var userApplication = Applications.findOne({}, {'status': 1, 'extend': 1});
 
-        if (moment().isAfter(Meteor.settings.public.applicationDeadline)) {
-        
+        if (moment().isAfter(Meteor.settings.public.applicationDeadline) && userApplication.status !== 'approved') {
```

Comment:
The next if also checks if after deadline. Could these 3 conditionals just be in 1?